MicroProfile-JWT-Auth/tck-arquillian-extension/pom.xml

```diff
@@ -83,7 +83,7 @@
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>=[5.193,)</value>
+                    <value>>=[5.193,)</value>
```

Comment:
This is invalid XML, you have to use CDATA or `&gt;` (hrml entity)

Reply:
Syntax is correct based on examples in base repo for extension: https://github.com/payara/ecosystem-maven-regex-profile-activation-extension
Compiles correctly through maven and all profiles activate as required

Reply:
No, this is definetely not correct syntax. If profile activates, then it is due do something else, as version range activator is only active when first character is `=`

Reply:
Hmm strange I'll have to look into that more - my bad though I misread the integration tests, could we get some proper documentation for it written up to avoid potential for confusion in future?

Reply:
Documentation requested.

Reply:
If Maven really parsed this XML without an exception, I would call it a miracle.
See http://xml.silmaril.ie/specials.html or https://www.w3schools.com/xml/xml_syntax.asp